src/main/java/appeng/client/render/effects/AssemblerFX.java

```diff
@@ -32,10 +32,10 @@
 public class AssemblerFX extends EntityFX
 {
 
-	final IAEItemStack item;
-	final EntityFloatingItem fi;
-	final float speed;
-	float time = 0;
+	private final IAEItemStack item;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "item" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)